data/network/src/commonTest/kotlin/social/androiddev/common/network/MastodonApiTests.kt

```diff
@@ -18,11 +18,15 @@ import kotlin.test.assertTrue
 import kotlinx.coroutines.runBlocking
 import kotlinx.serialization.json.Json
 import kotlin.test.Test
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
 
+@OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
Suggest adding this to kotlin compiler args in certain modules so we don't have to specify it all over the place

Reply:
I tried, but it looks like it is not available in KMP. I'll investigate later and do a PR if it is possible.